deepdive1/README.md

```diff
@@ -0,0 +1,371 @@
+# Deep Dive 1 :ocean: :swimming_woman: - Testing
+Testing your app is important for making sure your app functions as expected. Today we'll specifically be talking about Unit Testing, which is a form of automated testing. As the name implies, unit testing is where you write tests small units of your code. For example, lets say I want to test that the fibonacci function I wrote actually works. In plain Java it might look like this:
+
+```
```

Comment:
You can write `java` here after the backticks to do syntax highlighting, as well as with every other java code snippet below